File: GL_DotNetFullStack_Project/Models/Project.cs
using System;

namespace GL_DotNetFullStack_Project.Models
{
    public class Project : IBaseEntity
    {
        public int ID { get; set; }
        public string Name { get; set; }
        public string Detail { get; set; }
        public DateTime CreatedOn { get; set; } 
    }
}

File: gl_projectmanagement_app/src/components/Projects/Project.js
import React from 'react';

const Project=(props)=>{
    //EC6
    const { ID, Name, Detail, CreatedOn} = props.data;

    return(
    <tr>
        <td>{ID}</td>
        <td>{Name}</td>
        <td>{Detail}</td>
        <td>{CreatedOn}</td>
    </tr>
    );
}

export default Project;